### PR TITLE
Initial SAP BAPI GoodsMovementCreate Template

### PR DESCRIPTION
**Pull request by bdelangh**
I added a sample xml file and http test file so customer can use this to test the LogicApp. These files will need to be changed according to customer SAP setup

### CODE REVIEW

**Comment by Elaina-Lee** [comment]
Hi @bdelangh, I looked over the PR and noticed that a couple parts (including the connections) is not following our guidelines. I updated our validation test to help guide how to follow our guidelines. Would you be able to merge the latest main? Thank you!

**Comment by bdelangh** [comment]
SAP Connection in manifest json updated.
 "sap_#workflowname#": {
            "connectorId": "/serviceProviders/sap",
            "kind": "inapp"
        }

File: SAP-BAPI-GoodsMovementCreate/workflow.json [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                            "type": "Response"
+                        },
+                        "[BAPI]_Roll_back_transaction": {
+                            "inputs": {
+                                "parameters": {
+                                    "closeSession": true,
+                                    "sessionId": "@body('[BAPI_-_RFC]_Create_stateful_session')?['sessionId']"
+                                },
+                                "serviceProviderConfiguration": {
+                                    "connectionName": "sap",
```
Comment (Elaina-Lee) [inline SAP-BAPI-GoodsMovementCreate/workflow.json:64]: These will need to be changed accordingly as well. I approved the workflow.json to be a part of it, so the test output now suggests changes. For example,

 "referenceName": "sap" => "referenceName": "sap_#workflowname#"

File: SAP-BAPI-GoodsMovementCreate/manifest.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "skus": [
+        "standard"
+    ],
+    "kinds": [
+        "stateless"
+    ],
+    "details": {
+        "By": "Microsoft",
+        "Type": "Workflow",
+        "Trigger": "When_a_HTTP_request_is_received"
```
Comment (Elaina-Lee) [inline SAP-BAPI-GoodsMovementCreate/manifest.json:17]: This would have to be "Request" instead of name of the trigger. I will update this

**Comment by Elaina-Lee** [comment]
Other than the change requested above, since this includes artifacts that are not supported at the moment, we will need to add an additional disclaimer in prerequisite. For reference, Business Tracking template in the repo is following this format. Thanks!

**Pull request closed without merging**